dash/dash.py

```diff
@@ -87,9 +87,8 @@ class _NoUpdate(object):
 # pylint: disable=too-many-instance-attributes
 # pylint: disable=too-many-arguments, too-many-locals
 class Dash(object):
-    """
-    Dash is a framework for building analytical web applications.
-    No JavaScript required.
+    """Dash is a framework for building analytical web applications. No
+    JavaScript required.
```

Comment:
"No JavaScript required" loses a bit of its punch with the "No" off on a separate line 😏